### DISCUSSION TITLE
Retention days not correctly displayed in log

### DISCUSSION THREAD

**Discussion by hell-g** [offen/docker-volume-backup#299] [Q&A] [answered]
Hi there,
I am using pruning to keep the number of backups limited. This is working as expected. I configured my retention days with `BACKUP_RETENTION_DAYS: "14"` (inside an Ansible role, therefore the ":") and the pruning is following this. Just the output in the log is a bit misleading. For instance from today:
`time=2023-11-04T01:01:09.538+01:00 level=INFO msg="Pruned 1 out of 19 backups as their age exceeded the configured retention period of 0 days." storage=Local`
As mentioned above, I would expect that it says here `...retention period of 14 days`. Anybody else noticing this or am I misunderstanding this?

**Comment by m90** [comment]
This is a bug and now tracked in #300

**Comment by m90** [comment, marked as answer]
This is fixed in [v2.33.2](https://github.com/offen/docker-volume-backup/releases/tag/v2.33.2)
